FIAT/expansions.py

```diff
@@ -416,18 +505,104 @@ def __init__(self, ref_el, **kwargs):
         super(TetrahedronExpansionSet, self).__init__(ref_el, **kwargs)
 
 
-def polynomial_dimension(ref_el, degree):
+def polynomial_dimension(ref_el, n, continuity=None):
```

Comment:
element --> complex.  continuity only affects the dimension if you've subdivided.